man/journalctl.xml

```diff
@@ -723,13 +723,12 @@
         <term><option>--vacuum-time=</option></term>
         <term><option>--vacuum-files=</option></term>
 
-        <listitem><para>Removes the oldest archived journal files until the disk space they use falls below
```

Comment:
Why is the note about oldest archived journal being removed? Is this right @DaanDeMeyer ?

Reply:
It is replaced with the phrase "Reduce journal files to the size specified " below.

Reply:
Removing and reducing are not the same thing though. Which one is it?

Reply:
`--vacuum-size` reduces size. `--vacuum-time` removes oldest entries.

Reply:
@bluca I placed original description back and split the text for readability.